chart/skywalking/values.yaml

```diff
@@ -38,8 +38,10 @@ oap:
     pullPolicy: IfNotPresent
   storageType: null
   ports:
+    # add more ports here if you need
     grpc: 11800
     rest: 12800
+    zabbix: 10051
```

Comment:
Can we make this as an example comment? I think it is not very widely used, my suggestion is 

```
# add more ports here if you need, for example
# zabbix: 10051
```

Reply:
make sense